src/vaibhav/PrimeNumberAddition.java

```diff
@@ -0,0 +1,44 @@
+package vaibhav;
+
+//This program will find prime array and sum and average of all elements
+
+public class PrimeNumberAddition {
+	
+	static double finalCount=0;
+	
+	public static int[] primeNumber() {
+		int i,j,k=0;
+		int [] primeArray=new int[100];
```

Comment:
If tomorrow some one gave range more then 100 that time this will not work. instead of updating in array and then you are doing sum of those array element, better to sum in if condition it self, try to avoid array during if you can handle this scenario in single method, mostly in interview.